DOC-852: Credentials overhaul A-D (#2086)
n8n-io/n8n-docs: docs/integrations/builtin/credentials/customerio.md

## Using API key

To configure this credential, you'll need:

- A **Tracking API Key**: For use with the [Track API](https://customer.io/docs/api/track/){:target=_blank .external-link} at `https://track.customer.io/api/v1/`. See the FAQs below for more details.
- Your **Region**: Customer.io uses different API subdomains depending on the region you select. Options include:
    - **Global region**: Keeps the default URLs for both APIs; for use in all non-EU countries/regions.
    - **EU region**: Adjusts the Track API subdomain to `track-eu` and the App API subdomain to `api-eu`; only use this if you are in the EU.
- A **Tracking Site ID**: Required with your **Tracking API Key**
- An **App API Key**: For use with the [App API](https://customer.io/docs/api/app/){:target=_blank .external-link} at `https://api.customer.io/v1/api/`. See the FAQs below for more details.

Refer to the [Customer.io Finding and managing your API credentials documentation](https://customer.io/docs/accounts-and-workspaces/managing-credentials/){:target=_blank .external-link} for instructions on creating both Tracking API and App API keys.

> ## FAQs

Reviewer: ```suggestion

```

> ### Why do I need a Tracking API Key and an App API Key?

Reviewer: ```suggestion
## Why you need a Tracking API Key and an App API Key
```

Reviewer: Trying to get away from FAQ sections
